Lib/test/test_concurrent_futures/test_process_pool.py

```diff
@@ -168,6 +168,24 @@ def test_max_tasks_per_child(self):
 
         executor.shutdown()
 
+    def test_max_tasks_per_child_with_fast_submit(self):
+        # gh-115634:
+        # If many tasks are submitted quickly, the idle worker count
+        # would be wrong so the new worker won't be spawned.
+
+        context = self.get_context()
+        if context.get_start_method(allow_none=False) == "fork":
+            raise unittest.SkipTest("Incompatible with the fork start method.")
```

Comment:
Why not compatible with fork()?

Reply:
Ah, ok, can you enhance the skip message a bit then? Something like `max_tasks_per_child is incompatible with the fork start method`.

Reply:
The existing tests are using the exact same skip message:

```python
    def test_idle_process_reuse_one(self):
        executor = self.executor
        assert executor._max_workers >= 4
        if self.get_context().get_start_method(allow_none=False) == "fork":
            raise unittest.SkipTest("Incompatible with the fork start method.")
        executor.submit(mul, 21, 2).result()
        executor.submit(mul, 6, 7).result()
        executor.submit(mul, 3, 14).result()
        self.assertEqual(len(executor._processes), 1)
```

Do you want me to change those too? There are at least three of them.

Reply:
They were added with this feature to lazily spawn worker processes.

Reply:
Hmm, fair enough.